codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/CodegenVisitor.kt

```diff
@@ -73,19 +76,40 @@ class CodegenVisitor(context: PluginContext, private val codegenDecorator: RustC
         httpGenerator = protocolGenerator.buildProtocolGenerator(codegenContext)
     }
 
+    /** Base model transformation applied to all services
+     *
+     * - Add `Box<T>` to recursive shapes as necessary
+     * - Normalize the `message` field on errors
+     * - NormalizeOperations by ensuring every operation has an input & output shape
+     * - Drop unsupported event stream operations from the model
+     * - Normalize event stream operations
+     */
     private fun baselineTransform(model: Model) =
         model.let(RecursiveShapeBoxer::transform)
             .letIf(settings.codegenConfig.addMessageToErrors, AddErrorMessage::transform)
             .let(OperationNormalizer::transform)
             .let { RemoveEventStreamOperations.transform(it, settings) }
             .let(EventStreamNormalizer::transform)
```

Comment:
I think these will be easier to maintain if we inline them over each `.let(...)`